### ISSUE TITLE
Tokenize It - Stake proof of ownership of tangible assets to the blockchain.

### ISSUE THREAD

**Issue by MCDForm** [district0x/district-proposals#178]
# Name: 
Tokenize It
# Purpose: 
Tokenize tangible assets for proof of ownership, trades and resale.

A basic district that allows users to tokenize tangible assets to allow for proof of ownership, and transaction history staked to the blockchain to better support legitimate transfers of products over secondary markets.
# Description:
There are many firms developing technology to tokenize tangible assets but they are mostly focused on new products and expensive products.  Others companies such as Chronicled sell hardware solutions to “tokenize” objects that require using centralized apps to retrieve product information.

This district will make tokenizing accessible to everyone using methods similar to uploading and purchasing a picture from Snapfish and will store product and ownership information and history using IPFS and the Ethereum blockchain.
# Case use:
The secondary market for sneakers is currently worth over $1 Billion.  Many sneaker sales are handled face to face but the bulk are handled over the internet.  Some rely on trading reputation and others rely on authentication services (stockx, GOAT).  Those willing to take the biggest risk rely on nothing but good faith.

This district will allow the user to purchase a sneaker from a store/website, upload a  collection of photos, receipts, and files that show proof of ownership and then mint the token using the Memefactory infrastructure.

Buyers will be able to verify that the sneaker they purchased is from the previous owner and that the same object (or clone) has not been sold twice or is being sold by a non-owner.  
# Notes:
1. This case use can be translated to many popular resale markets including records, purses, musical instruments, baseball cards, and anything object that relies on authenticity for value.  

1. When attempting to tokenize tangible assets the risk of fraud cannot be 100% mitigated without hardware or tokenization at origin.  Tokenizing at origin is impossible for items already on the secondary market. 

1. Increased fraud prevention could be added by acquiring a single authentication certificate for the item that is linked to its proof of ownership.  Objects with serial numbers or UID (high end purses) can be slightly more secure as well by utilizing their specific UIDs in their tokenization.

1. This would also allow users to turn tangible assets into investment vehicles by creating shares of an object that normally could not be divided or is too inexpensive to draft paper contract for.

1. This district could be easily employed after Memefactory.  It could be adapted to multiple types of marketplace districts or it could be a hub for tokenization of tangible assets which are then sold in marketplace districts.  Proof of ownership and network wide repuation could thoroughly reduce the risk involved for the buyer and seller.

1. A printable, postable code (QR Code, etc) that links directly to the item and information would be valuable for many collectors.

*In the case of this proposal, "tokenize" is used as a generic term for staking to the blockchain.  The tokenzing process may require minting a new token, a hash, UID, or a smart contract alone or in concert with each other.  The process will be much clearer with the release of Memefactory.

Eth: 0x1c63a09C62558B2A614f48e7495291f02f99AC40

**Comment by Bradymck** [comment]
Proposal machine is your new name lol

Love your ideas man. You really put a lot of effort into these. 

One question, how would you classify this? I added the tokenization tag but if you have a better idea that may help highlight similar ideas for collaboration let me know and I will add.

**Comment by MCDForm** [comment]
@Bradymck I'd maybe categorize this one as a service or even template but the tokenization tag is the most accurate.  

I post all my proposal for two reasons.  One is that maybe someone will want to collaborate, either on a proposal or on bringing my idea to light after this proposal process is complete and secondly I love the thought process that goes behind making proposals.  It starts as an idea but slowly comes alive in my notebook until it forms a full fledged proposal.  This one came about when I was  working on some shoes that I want to "tokenize" but have no way to really do it.  I think this one could also be used in concert with many other districts in the future.

**Comment by Bradymck** [comment]
I think I may actually tokenize my guitars as a fun project. Correct me if wrong, a chain of custody is essentially what you are describing. 

Though I feel like some kind of follow through with the next person in the chain is the issue. An easy identity system needs to be developed. I like what Civic is doing but I have mixed emotions about blockchain identity. 

I would hate to see blockchain identity turn into a barrier to entry for services if a person is stuck in a less than ideal life situation. It's a double edged sword.

Take for instance a compliance program at your bank that has cut you off after using your blockchain identity that is tied to your real identity. You now are completely stripped of access to services even if not doing something illegal. Heck I was cut off from Paypal at one time just for using the word bitcoin. Mind you this was a few years ago.

Take for instance [operation choke point](https://en.wikipedia.org/wiki/Operation_Choke_Point). If you are cut off from doing anything on this list, you are stripped of access for doing something that is not even illegal. 

It's said that operation choke point has ceased operation. However a lot of financial institutions will not do business with these types of businesses nor the people involved. So it's still in place, even if it's not actively a government program anymore. It has engrained in the private sector and will remain so. 

Though I digress. I do feel we are in an identity crises of the modern day. So many conflicting issues these days when it comes to identity. This seems like a problem that needs to be solved soon or a chain of custody will be tough to pull off.

What are your thoughts? Also please clarify if I misunderstand the purpose of why you wrote this proposal and how this would function as a market. I feel I may be missing something.

**Comment by MCDForm** [comment]
Yes basically a chain of custody.  This is easier on a new product where the manufacturer can include UID but  2nd hand goods will be a little tougher.

What you're describing sounds a lot like how they are using Resident Registration numbers in South Korea.  Your online existence is tied to your real life existence and your reputation follows you.  This is the opposite of decentralization and allows all sorts of censorship.

Chain of custody will be really hard to exist without holding the physical item (like evidence).  I've seen a few companies digging into it like verisart and https://www.chronicled.com/ (I like the temp monitor for food).  Without a third party to verify that a 2nd hand good is legit then this district is really based on trust.  This is why it would allow some proof of ownership docs to be included in the process.  If your product was authenticated by an expert you could include that.

The market aspect is actually more of a service.  Charging a small fee to tokenize and store some of the proof of ownership docs.  It could easily be ported to multiple types of marketplaces but I didn't want to limit it to a specific type in the proposal.

**Comment by dliuproduction** [comment]
To make this district function as a market, we could have two types of users, Traders and Authenticators.

The buyer stakes some amount of token while the seller stakes some amount of tokens representing the valuation of their sneakers.

If either parties back out of a trade, the contract deducts their token and awards the other party.

While their token is being held as deposit, this large token pool can be used to incentivize Authenticators to review sneaker postings, photos, serial numbers, conditions, and documents. 

Reviews can also happen in forms of staking, where those that stake with the majority get a small percentage of tokens from the pool as reward. 

This solves trust and authenticity problems of both trading face to face and online.

These are my rough thoughts, but I have had this idea for a long time and have just come across your proposal.

Let me know what you think.

**Comment by Bradymck** [comment]
Miss you @MCDForm 

You had vision in this space most people don't. Most your ideas are either built or easily able to be built now. Cheers man, hope you're well. 

Also, if you ever want to hop in the community again, I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the slightly canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
